### DISCUSSION TITLE
Deformable elastic bodies

### DISCUSSION THREAD

**Discussion by shubhamt2897** [google-deepmind/mujoco#1789] [Asking for Help]
I have been trying to simulate objects like hoses which deform during robot's motion, in the default code damping and material properties is what I am looking into right now. I saw the old video of 2.3 for an elasticity plugin but couldn't find it . Can anyone direct me in the right direction, if they have worked on similar applications?

**Comment by quagla** [comment]
Hi, the related examples are all here https://github.com/google-deepmind/mujoco/tree/main/model/plugin/elasticity
